tests/formdesigner.lock.js

```diff
@@ -194,7 +194,7 @@ require([
                 btn.click();
                 assert.equal(
                     btn.next().find('li:contains("Cannot Change Question Type")').length,
-                    !bool);
+                    +!bool);
```

Comment:
This was the only change necessary when I switched `assert.equal` and `assert.notEqual` to use strict equality. `+` converts a boolean value to an integer.